dist/keysim.js

```diff
@@ -1,123 +1,110 @@
-(function (global, factory) {
```

Comment:
So, I get it now. Stuff don't have to be checked in to git for them to be included in the npm package. They just have to be in the directory, right?